### ISSUE TITLE
pretrianed model

### ISSUE THREAD

**Issue by Azure985** [NVlabs/planercnn#10]
Can you share your pre-training model elsewhere? There is no way to connect to the dropbox in China.thx!

**Comment by dreamPoet** [comment]
For mask rcnn coco, I downloaded it from mask rcnn github repo and it worked.

**Comment by zly9120** [comment]
So have you got the pretrained model? Can you share that to me? thank you. The dropbox link is not available.
